101-dashing-Ecommerce/index.html

```diff
@@ -0,0 +1,107 @@
+    <link rel="stylesheet" href="styles/newsletter.css">
+    <link rel="stylesheet" href="styles/footer.css">
+    <link rel="stylesheet" href="styles/responsive.css">
+    <link rel="stylesheet" href="styles/animations.css">
+</head>
+<body>
+    <header class="header">
+        <div class="container">
+            <div class="logo">CoolShop</div>
+            <nav class="navbar">
+                <ul>
+                    <li><a href="#">Home</a></li>
+                    <li><a href="#">Shop</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </nav>
+            <div class="cart">
+                <a href="#">Cart (0)</a>
+            </div>
+        </div>
+    </header>
+
+    <section class="hero">
+        <div class="container">
+            <h1>Welcome to CoolShop</h1>
+            <p>Your one-stop shop for all things cool!</p>
+            <a href="#" class="btn">Shop Now</a>
+        </div>
+    </section>
+
+    <section class="products">
+        <div class="container">
+            <h2>Featured Products</h2>
+            <div class="product-grid">
+                <div class="product-card">
+                    <img src="https://placehold.co/600x400/EEE/31343C" alt="Product 1">
+                    <h3>Product 1</h3>
+                    <p>$19.99</p>
+                </div>
+                <div class="product-card">
+                    <img src="https://placehold.co/600x400/EEE/31343C" alt="Product 2">
+                    <h3>Product 2</h3>
+                    <p>$29.99</p>
+                </div>
+                <div class="product-card">
+                    <img src="https://placehold.co/600x400/EEE/31343C" alt="Product 3">
+                    <h3>Product 3</h3>
+                    <p>$39.99</p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="testimonials">
+        <div class="container">
+            <h2>What Our Customers Say</h2>
+            <div class="testimonial-slider">
+                <div class="testimonial">
+                    <p>"This is the best shop ever!"</p>
+                    <h3>- Happy Customer</h3>
+                </div>
+                <div class="testimonial">
+                    <p>"Amazing products and great service."</p>
+                    <h3>- Satisfied Client</h3>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <h1>Contact Us!</h1>
+
+    <section class="newsletter">
+        <div class="container">
+            <h2>Subscribe to our Newsletter</h2>
+            <form>
+                <input type="email" placeholder="Enter your email">
+                <button type="submit">Subscribe</button>
+            </form>
+        </div>
+    </section>
+
+    <footer class="footer">
+        <div class="container">
+            <p>&copy; 2024 CoolShop. All rights reserved. Work And Design By Anas.</p>
```

Comment:
*2024* should be in a <span> element with id="year" and set it's content automatically to the current year with JS. You don't want to update it every year :)